sup3r/preprocessing/data_handling/dual_data_handling.py

```diff
@@ -253,18 +251,15 @@ def normalize(self, means=None, stds=None, max_workers=None):
         super().normalize(means=means, stds=stds,
                           features=self.lr_dh.features,
                           max_workers=self.lr_dh.norm_workers)
+        self.lr_dh.normalize(means=means, stds=stds,
+                             features=self.lr_dh.features,
+                             max_workers=self.lr_dh.norm_workers)
         logger.info('Normalizing high resolution data features='
                     f'{self.hr_dh.features}')
         self.hr_dh.normalize(means=means, stds=stds,
                              features=self.hr_dh.features,
                              max_workers=self.hr_dh.norm_workers)
 
-        # need to normalize data attribute arrays in addition to handlers
-        mean_arr = np.array([means[fn] for fn in self.lr_dh.features])
-        std_arr = np.array([stds[fn] for fn in self.lr_dh.features])
-        self.lr_data = (self.lr_data - mean_arr) / std_arr
-        self.lr_data = self.lr_data.astype(np.float32)
-
         if id(self.hr_data.base) != id(self.hr_dh.data):
```

Comment:
Well I think it would just get updated after `val_data` is set since the `val_split` method just updates the `hr_dh.data` attribute, but I agree it might be confusing. Theres also the option of just using the `_normalize` method to directory normalize hr_data but since you want everything normalized maybe this current way makes the most sense.